prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -89,9 +89,18 @@ public async Task SendSiteRegistrationSubmissionAsync(int siteId, int businessLi
             await _emailDocumentService.SaveSiteRegistrationReview(siteId, siteRegReviewPdf);
         }
 
-        public async Task SendSiteReviewedNotificationAsync(int siteId, string note)
+        public async Task SendHealthAuthoritySiteRegistrationSubmissionAsync(int healthAuthoritySiteId)
         {
+            var email = await _emailRenderingService.RenderSiteRegistrationSubmissionEmailAsync(new LinkedEmailViewModel(null), CareSettingType.HealthAuthority);
```

Comment:
So one thing about the service is it sends the email to HIBC and Support when the ticket just asked for a CC to support since this is not actionable by them. I think its worth updating that in "RenderSiteRegistrationSubmissionEmailAsync" so when its not Comm pharm it CC's support.

Reply:
Updated to CC

Reply:
Closed as discussed